src/js/reducers.js

```diff
@@ -561,12 +581,74 @@ function system(state = {}, action) {
     }
 }
 
+function appStore(state = {}, action) {
+    switch (action.type) {
+        case Actions.UPDATE_AVAILABLE_APPSTORE_APPS:
+            var newState = { ...state };
+            newState.availableApps = action.availableApps;
+            newState.installedApps = state.installedApps ? state.installedApps.map((app) => {
+                var appDirEntry = action.availableApps.find(x => x.policyAppID === app.policyAppID);
+                return appDirEntry ? Object.assign(appDirEntry, app) : app;
+            }) : [];
+            return newState;
+        case Actions.UPDATE_INSTALLED_APPSTORE_APPS:
+            var newState = { ...state };
+            newState.installedApps = action.installedApps.map((app) => {
+                var appDirEntry = newState.availableApps ? newState.availableApps.find(x => x.policyAppID === app.policyAppID) : {};
+                appDirEntry.runningAppId = 0;
+                return Object.assign(appDirEntry, app);
+            });
+            return newState;
+        case Actions.ADD_APP_PENDING_SET_APP_PROPERTIES:
+            var newState = { ...state };
+            if(!newState.appsPendingSetAppProperties){
```

Comment:
```suggestion
            if (!newState.appsPendingSetAppProperties) {
```

Reply:
Fixed in https://github.com/smartdevicelink/generic_hmi/pull/240/commits/4c6c9dbd8def6a6b0124c61c001b71e5e2dd6fc4